MozillaFoundation/foundation.mozilla.org
# Remove localized from description in blog_card template

# Description

This PR removes `.localized` from blog_card.html template, reducing the number of `.localized ` attributes in the codebase, and still keeping the information translated and displayed properly thanks to the `localize_queryset` function already created on 
`/networkapi/wagtailpages/utils.py`.

Link to sample test page: https://foundation-s-tp1-100-re-cee0m8.herokuapp.com/en/
Related PRs/issues: https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10007

# How to test
- Pull and checkout this branch into your local env, and start app.
- Go to [blogs section](http://localhost:8000/en/blog/)
- Change the locale slug from "en" to any other locale available.
- Make sure the blog card description is still displayed properly and translated accordingly.

Image for reference:
![image](https://github.com/user-attachments/assets/80e17822-c703-4cf5-8793-7b55dafbc503)

### Caveats:
- In case there is not any current translation, the meta description will remain the same. So you can go to the CMS > open the blog page that will be tested > go to promote section > switch locale > add translation > publish.

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**

**Merge Method**
**💡❗Remember to use squash merge when merging non-feature branches into `main`**

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1280)
